src/main/java/me/botsko/prism/database/sql/SqlPrismDataSourceUpdater.java

```diff
@@ -91,4 +91,19 @@ public void v6_to_v7() {
             dataSource.handleDataSourceException(e);
         }
     }
+
+    @Override
+    public void v7_to_v8() {
+        String query;
+        try (
+            Connection conn = dataSource.getConnection();
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['try resource' has incorrect indentation level 12, expected level should be 14.](https://app.codacy.com/gh/AddstarMC/Prism-Bukkit/pullRequest?prid=5609620)